web/src/utils/dashboard/convertSQLData.ts

```
@@ -555,6 +562,190 @@ export const convertSQLData = async (
+          index >=
+          gridData.gridNoOfRow * gridData.gridNoOfCol - gridData.gridNoOfCol;
+      } else {
+        showAxisLabel = index % gridData.gridNoOfCol === 0;
+      }
+
+      // Here we are setting the axis label properties, if showAxisLabel is false then we are hiding the axis label
+      it.nameGap = showAxisLabel ? yAxisNameGap : 0;
+      it.axisLabel.margin = showAxisLabel ? 5 : 0;
+      it.axisLabel.fontSize = showAxisLabel ? 12 : 10;
+      it.nameTextStyle.fontSize = 12;
+      it.axisLabel.show = showAxisLabel ? true : false;
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)